src/forms/current-benefits.js

```diff
@@ -20,27 +20,26 @@ import {
 */
 const CurrentBenefitsContent = (props) => {
 
-	var origin = props.origin,
-      client = origin.pageState;
+  var client = props.pageState;
```

Comment:
You instantiate `client` here, but doesn't seem to get used in the following lines. I personally do like the idea of this declaration up here to avoid `props.client` in all the other spots, so I'd go with that. Also, does `props` have both a `client` and a `pageState`?

Reply:
I probably just forgot to remove that. `props` doesn't have have a `pageState` anymore, so I can probably just declare `client = props.client`

Reply:
Or we can use ES6 destructuring like so:

`const CurrentBenefitsContent = ({ client, storeChecked })`